.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-linux`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.DiagnosticSource.dll; SideIndicator=<=}, @{InputObject=System.Net.NameResolution.dll; SideIndicator=<=}, @{InputObject=System.Private.Xml.Linq.dll; SideIndicator=<=}, @{InputObject=System.Xml.XDocument.dll; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 124
124:         $diffs | Should -BeExactly $null
```
Using of ternary operator.Ternary expression '{ $($p = Get-Process -Id $PID; $p.Name -eq 'pwsh') ? 'Core' : 'Desktop' }' should generate correct results
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   7
Strings differ at index 0.
Expected: 'Core'
But was:  'Desktop'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/TernaryOperator.Tests.ps1: line 56
56:         & $Script | Should -BeExactly $ExpectedValue
```
Using of ternary operator.Ternary expression '{ $IsCoreCLR -and (Get-Process -Id $PID).Name -eq 'pwsh' ? 'Core' : 'Desktop' }' should generate correct results
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   7
Strings differ at index 0.
Expected: 'Core'
But was:  'Desktop'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/TernaryOperator.Tests.ps1: line 56
56:         & $Script | Should -BeExactly $ExpectedValue
```
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.269696349876346.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1932
1932:             $pathologicalRatio | Should -BeGreaterThan 5
```